workspaces/demo-app/src/components/cogito-attestations/CogitoAttestations.js

```diff
@@ -22,7 +22,7 @@ const ShowAttestation = () => (
   <Centered css={{ padding: '10px', marginRight: '50px', backgroundColor: 'white' }}>
     <p>Scan to add a dummy attestation:</p>
     <CogitoQRCode
-      value='https://cogito.mobi/attestations/receive#A=email%3Atest.user%40philips.com'
+      url='https://cogito.mobi/attestations/receive#A=email%3Atest.user%40philips.com'
```

Comment:
This is indeed a good change. I would even propose to make it more descriptive and name it `connectUrl` as this is the vocabulary used by `telepath`. `telepath` has a method called `createConnectUrl` to get that url - it will be then more clear to the user _what_ kind of url is intended here.

Reply:
Renamed to connectUrl